pkg/core/blockchain.go

```diff
@@ -704,6 +705,62 @@ func (bc *Blockchain) tryRunGC(old uint32) time.Duration {
 		tgtBlock /= int64(bc.config.GarbageCollectionPeriod)
 		tgtBlock *= int64(bc.config.GarbageCollectionPeriod)
 		dur = bc.stateRoot.GC(uint32(tgtBlock), bc.store)
+		dur += bc.removeOldTransfers(uint32(tgtBlock))
+	}
+	return dur
+}
+
+func (bc *Blockchain) removeOldTransfers(index uint32) time.Duration {
```

Comment:
We also have related TODO inside the `blockchain.go`:
```
// TODO: remove NEP-17 transfers and NEP-17 transfer info for genesis block, #2096 related.
```
I think we may safely remove this todo and leave that code as is, because outdated NEP17 transfers for genesis block will be removed in a regular manner by `removeOldTransfers` in the subsequent GC run.

Reply:
It just came to my mind, GC can be turned off with P2PStateExcangeExtensions on, so we still need to take care of outdated genesis transfers manually during statejump.

Reply:
Well, if GC is off then we store everything and storing 2 transfers for the genesis is no big deal. If it's enabled it'll do its job (although I'm not sure there is a lot of activity on these addresses after 1M blocks for example).

Reply:
I've added a patch to just always drop it.